sdcm/cluster_k8s/__init__.py

```diff
@@ -2975,60 +2998,71 @@ def _get_docker_image(self):
     def add_nodes(self,
                   count: int,
                   ec2_user_data: str = "",
-                  dc_idx: int = 0,
+                  # NOTE: 'dc_idx=None' means 'create %count% nodes on each K8S cluster'
```

Comment:
This is how other backends behave ??

If not, why do we want this behavior?

Reply:
It is not the same. In VMs case we don't have additional sub-layer as `K8S clusters`.
It is just flexibility approach.
When/if we support creation of different number of nodes in different K8S clusters we will not use this approach.